pkg/apis/dashboard_manifest.go

```diff
@@ -11,8 +11,6 @@ import (
 	"github.com/grafana/grafana-app-sdk/app"
 )
 
-var ()
```

Comment:
Is it needed? 🤔

Reply:
This was a pre-commit hook lint change. I can't seem to commit without it making the change (unless I disable the hook)!